_emails/newsletter-2020-10.html

```diff
@@ -0,0 +1,95 @@
+1. Languages & Tools (History)
+2. Structure
+3. Delivery
+4. Practical Workshop
+
+We'll be running the sessions Tuesday, 3rd November from 10:00am (London time). If you would like to be added to the invite please let us know in the [#origami-support](https://financialtimes.slack.com/messages/origami-support) Slack channel or with an email to origami.support@ft.com. 😊
+{% endcapture %} {% include email/markdown.html content=text %}
+
+<!-- Thing 2 -->
+{% include email/h3.html content="o-tracking Major Release" %} {% capture text %}
+You may know from our previous issues that the Origami team had taken on stewardship of `o-tracking` (there is a [technology proposal to find new owners](https://github.com/Financial-Times/technology-proposals/issues/51)). Well, we've just released a new major version (v3) with some big improvements:
+-  Using cookies as a storage mechanism for queuing events has been removed to improve site performance. The browser would send the tracking cookie (which ranged in size from 500 bytes to 2.5kb) with every request on a page, for example to fetch each image, slowing down each request and making page loading slower. At the cookie's largest size, and with over 100 requests made to render a site like ft.com, that adds up to a significant data saving 🎉
+- The Beacon API is used by default by all browsers which support it, helping ensure we don't miss data.
+- <abbr title="Application programming interface">API</abbr> changes have been made to improve usability and future maintainability, i.e. by reducing the public <abbr title="Application programming interface">API</abbr> and number of configuration options.
+- Documentation has been updated with more detail and examples.
+
+See the [o-tracking migration guide](https://github.com/Financial-Times/o-tracking/blob/master/MIGRATION.md#migrating-from-v2-to-v3) for more details.
+{% endcapture %} {% include email/markdown.html content=text %}
+
+<!-- Thing 3 -->
+{% include email/h3.html content="New Proposal To Drop Bower Support" %} {% capture text %}
+We have a new proposal this month to drop the [Bower package manager](https://bower.io/) from Origami components, for both developing and using components, in favour of the [npm](https://www.npmjs.com/) package manager.
+
+Why now? Bower allows us to maintain a flat dependency tree for frontend Origami components. What that means is Bower ensures we only ever including one version of a component. That is important as multiple versions of the same component, say too versions of a button, could conflict with each other causing unpredictable visuals or errors; it would also mean forcing a users web browser to download and render multiple versions, slowing down our websites. But npm v7, released this month, supports the automatic install of peer dependencies. It will finally allow us to maintain a flat Origami dependency tree without interfering with backend dependencies which don't have the same requirement.
```

Comment:
```suggestion
Why now? Bower allows us to maintain a flat dependency tree for front-end Origami components. What that means is Bower ensures we are only ever including one version of a component. That is important as multiple versions of the same component, say two versions of a button, could conflict with each other causing unpredictable visuals or errors; it would also mean forcing a users web browser to download and render multiple versions, slowing down our websites. But npm version 7, released this month, supports the automatic install of peer dependencies. It will finally allow us to maintain a flat Origami dependency tree without interfering with back-end dependencies which don't have the same requirement.
```